lib/ansible/modules/monitoring/grafana_datasource.py

```diff
@@ -92,6 +92,12 @@
     description:
       - The TLS CA certificate for self signed certificates.
       - Only used when C(tls_client_cert) and C(tls_client_key) are set.
+  tls_skip_verify:
+    required: false
```

Comment:
If it's not required, don't add this please.

Reply:
ok, removed